package/firmware/linux-firmware/intel.mk

```diff
@@ -199,6 +199,14 @@ define Package/iwlwifi-firmware-ax210/install
 endef
 $(eval $(call BuildPackage,iwlwifi-firmware-ax210))
 
+Package/iwlwifi-firmware-ax411 = $(call Package/firmware-default,Intel AX411 firmware)
+define Package/iwlwifi-firmware-ax411/install
+	$(INSTALL_DIR) $(1)/lib/firmware
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/iwlwifi-so-a0-gf4-a0-72.ucode $(1)/lib/firmware
```

Comment:
Please use a more recent version of this firmware. The wireless drivers from kernel 6.6 which we use support version 83. Please use `iwlwifi-so-a0-gf4-a0-83.ucode` instead.

Reply:
Sure thing.